### PR TITLE
Added logic to pass build flags when running the generator

### PR DESCRIPTION
**Pull request by max-mironov**
Hi, currently we don't have an option to pass build flags when running the generator while bootstrapping. 

`Easyjson` doesn't provide a way to inject build flag and failed when running on Jenkins machine configured to use go 1.13. due to the usage of modules and vendoring (we need to pass `-mod=vendor` flag when building on CI machine). 
Unfortunately this is not an option to upgrade to recent `Go` version at that moment - so it would be great to have an ability to pass required build flags if needed to avoid build failures.
 
Thank you fo building a great tool and hope that this PR could be merged!

### CODE REVIEW

File: bootstrap/bootstrap.go [hunk 1]
```diff
@@ -178,7 +182,13 @@ func (g *Generator) Run() error {
 		defer os.Remove(f.Name()) // will not remove after rename
 	}
 
-	cmd := exec.Command("go", "run", "-tags", g.BuildTags, filepath.Base(path))
+	execArgs := []string{"run"}
+	if g.GenBuildFlags != "" {
+		execArgs = append(execArgs, strings.Split(g.GenBuildFlags, buildArgsSeparator)...)
```
Comment (GoWebProd) [inline bootstrap/bootstrap.go:187]: But if you have

```
-gen_build_flags "-example 'text with space'"
```

**Comment by GoWebProd** [comment]
Hello. Thank you for your PR. But I have one comment for you.

File: bootstrap/bootstrap.go (hunk 1 above)
Comment (max-mironov) [reply bootstrap/bootstrap.go]: Hey @GoWebProd - thanks for quick review!
I've fixed the issue, please take a look.

```
bin/easyjson -all -gen_build_flags="-mod=mod -gccgoflags '-march=native -O3'" ./tests/data.go
Regexped flags [-mod=mod -gccgoflags '-march=native -O3']
Regexped flags len 3
[go run -mod=mod -gccgoflags '-march=native -O3' -tags  easyjson-bootstrap433561849.go]

bin/easyjson -all -gen_build_flags="-mod=mod" ./tests/data.go
Regexped flags [-mod=mod]
Regexped flags len 1
[go run -mod=mod -tags  easyjson-bootstrap252406791.go]

bin/easyjson -all -gen_build_flags="-mod=mod -x" ./tests/data.go
Regexped flags [-mod=mod -x]
Regexped flags len 2
[go run -mod=mod -x -tags  easyjson-bootstrap003937401.go]

bin/easyjson -all -gen_build_flags="" ./tests/data.go
[go run -tags  easyjson-bootstrap846570929.go]
```

**Comment by GoWebProd** [comment]
Thanks!

**Pull request merged**